tests/integration/package/test_package_command_image.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
Is this logging needed? I dont see it used in the codebase below

Reply:
Oops, I’m sure that’s left over from testing. Would you like me to remove it, or not touch this.PR after your approval?

Reply:
we should remove any unused imports